app/questionnaire/questionnaire_schema.py

```diff
@@ -224,6 +224,9 @@ def get_repeating_list_for_section(self, section_id):
     def get_repeating_title_for_section(self, section_id):
         return self._sections_by_id.get(section_id).get("repeat", {}).get("title")
 
+    def get_repeating_page_title_for_section(self, section_id):
+        return self._sections_by_id.get(section_id).get("repeat", {}).get("page_title")
```

Comment:
We should have a `get_repeat_for_section` method given the `self._sections_by_id.get(section_id).get("repeat", {})` is used in three places